includes/core-changes/aspnetcore/3.0/add-authz-assembly-change.md

```diff
@@ -0,0 +1,39 @@
+### AddAuthorization overload moved to a different assembly
+
+The core `AddAuthorization` methods that used to reside in `Microsoft.AspNetCore.Authorization` were renamed to `AddAuthorizationCore`. The old `AddAuthorization` methods still exist, but are in the `Microsoft.AspNetCore.Authorization.Policy` package instead. Apps using both methods should see no impact. Apps that weren't using the policy package must switch to using `AddAuthorizationCore`.
+
+#### Version introduced
+
+3.0
+
+#### Old behavior
+
+`AddAuthorization` methods existed in `Microsoft.AspNetCore.Authorization`.
+
+#### New behavior
+
+`AddAuthorization` methods exist in `Microsoft.AspNetCore.Authorization.Policy`. `AddAuthorizationCore` is the new name for the old methods.
```

Comment:
`AddAuthorizationCore` to me implies a less heavy dependency like `AddMvcCore` does. If that pattern of naming has changed it is confusing for end users IMO.  For my use case I was about to use `AddAuthorizationCore` but actually I think from what the docs here say I need to use `AddAuthorization`

Reply:
Great feedback for @blowdart and @HaoK.

Reply:
The only difference between the two, is that AddAuthorization adds the policy evaluator as well (which is needed by Endpoint routing).  Its basically sugar which aggregates the two lower Add methods.

https://github.com/aspnet/AspNetCore/blob/master/src/Security/Authorization/Policy/src/PolicyServiceCollectionExtensions.cs#L55